bitnami/elasticsearch/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: elasticsearch
-version: 12.0.1
+version: 12.0.2
```

Comment:
You're adding new features (e.g. podlabels). Therefore, you'll have to bump the minor version

Reply:
Chart version bumped